src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java

```diff
@@ -23,6 +23,107 @@ public void testConstructor_defaultConstructor_fieldsShouldHaveCorrectDefaultVal
         assertTrue(rubricDetails.getRubricWeights().isEmpty());
     }
 
+    //V
+    @Test
+    public void testShouldChangesRequireResponseDeletion_emptyRubricChoices_shouldReturnTrue() {
+        FeedbackRubricQuestionDetails rubricDetails = new FeedbackRubricQuestionDetails();
+        rubricDetails.setRubricChoices(Arrays.asList("test-1", "test-2"));
+        FeedbackRubricQuestionDetails newrubricDetails = new FeedbackRubricQuestionDetails();
```

Comment:
```suggestion
        FeedbackRubricQuestionDetails newRubricDetails = new FeedbackRubricQuestionDetails();
```
Do make the same change to the other testcases.